pkg/core/resources/apis/mesh/mesh_helpers.go

```diff
@@ -36,3 +38,29 @@ func (m *MeshResource) GetTracingBackend(name string) *mesh_proto.TracingBackend
 	}
 	return backends[name]
 }
+
+// GetLoggingBackends will return logging backends as comma separated strings
+// if empty return default logging
+func (m *MeshResource) GetLoggingBackends() string {
+	var backends []string
+	for _, backend := range m.Spec.GetLogging().GetBackends() {
+		backends = append(backends, backend.GetName())
+	}
+	if len(backends) > 0 {
```

Comment:
is this if necessary? I think that `strings.Join(backends, ", ")` returns empty string when the array is empty